safe-global/safe-wallet-web
# Feat: add safe logo to header bar on mobile

## What it solves

Resolves #3932

## How this PR fixes it
- Adds small Safe logo on mobile viewports
- Removes the chain name for the network selector on mobile viewports

## How to test it
- Open a safe on a mobile screen size
- Check that the header contains the new logo which behaves the same as the larger logo.
- Check that the header is displayed correctly for all screen sizes.

## Screenshots
Disconnected: 
![image](https://github.com/user-attachments/assets/5adcebed-cd7e-4f73-b476-19efb667e0b6)

Connected:
![image](https://github.com/user-attachments/assets/3a39cfe0-0661-47a2-a050-a17275d172c4)


## Checklist
